### ISSUE TITLE
Using ExternalIP in devspace.yaml and devspace init

### ISSUE THREAD

**Issue by michalip** [devspace-cloud/devspace#565]
Hello, 

we can specify external IPs for services in K8s using the below yaml extract
Can we perform the same in devspace during the dialog of devspace init or at least edit the devspace.yaml for now ?

K8s yamp example for external IP
```
apiVersion: v1
kind: Service
metadata:
  name: REDUCTED
  namespace: REDUCTED
spec:
  externalIPs:
  - 10.X.X.X
  ports:
  - name: REDUCTED
```

Thank you 

**Is your feature request related to a problem?**  
Probably not but I cant find it in documentation so far

**Which solution do you suggest?**  
To follow normal K8s Yaml example

**Which alternative solutions exist?**  
manually expose the port using kubectl expose

**Additional context**  

/kind feature

**Comment by FabianKramm** [comment]
@michalip thanks for opening this issue! I'm not sure if I understand the problem correctly, do you want an option in devspace.yaml in the component deployment where you can specify an external ip? If you have a working service.yaml that looks like the one you have provided you can just use the config
```
deployments:
- name: my-deployment
   kubectl:
    manifests:
    - my-folder/service.yaml
```
to let devspace deploy it.

**Comment by michalip** [comment]
Hi @FabianKramm, yes I was asking to see if I can pass a line to devspace.yaml to indicate an externalIP but I will definitely try with the manifests as I do have a working K8s yaml already.
Thanks!!

**Comment by LukasGentele** [comment]
Generally, it is not recommended to use externalIPs because it impedes the sharing capabilities of the configuration (two users can generally not expose a service on the same IP:PORT combination). However, I believe we should be compliant with the k8s service spec for this issue, so I released v0.0.2 of the component chart and make the according changes in DevSpace CLI to account for this.

See PR: #584

**Comment by michalip** [comment]
Thanks @LukasGentele 
we can close this also
